pcdsutils/log.py

```diff
@@ -243,18 +247,31 @@ def configure_pcds_logging(
     return handler
 
 
-def validate_log_level(level) -> int:
+def validate_log_level(level: typing.Union[str, int]) -> int:
     '''
-    Return a int for level comparison
+    Return a logging level integer for level comparison.
```

Comment:
```suggestion
    Return a logging level integer for level comparison.
    
```

Not sure if it's really required by our style guide, but the docstring _feels_ squished

Reply:
I'm not sure what happened here, I copied this updated function over from hutch-python and the docstrings seem to have shrunk themselves

Reply:
Triple quotes also adjusted to match the norm